bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryParam.scala

```diff
@@ -80,6 +80,16 @@ object InvestigatorInfo {
   }
 }
 
+sealed case class PartnerInfo(name: String, hoursAllocated: Double)
```

Comment:
Why are these sealed? Since you can't (or shouldn't) extend case classes it seems unecessary